data/remnant/remnant outfits.txt

```diff
@@ -678,10 +678,10 @@ outfit "Bellows-Class Afterburner"
 	"outfit space" -7
 	"engine capacity" -7
 	"slowing resistance" 0.01
-	"afterburner thrust" 125.0
-	"afterburner fuel" 0.65
-	"afterburner energy" 6.0
-	"afterburner heat" 8.0
+	"afterburner thrust" 65
+	"afterburner fuel" 0.5
+	"afterburner energy" 6
+	"afterburner heat" 30
```

Comment:
this energy/heat ratio kinda makes it like a korath AB, thereby losing some of its uniqueness, no?
also do consider the remnants would invest a lot of ressources into making a good AB, I also think we should keep the same fuel and up the thrust back to 90
I'd also bring the energy usage up (say 9?) and the heat down (18?)